torch/onnx/symbolic_opset10.py

```diff
@@ -0,0 +1,123 @@
+import torch
+from torch.nn.modules.utils import _single, _pair, _triple
+import torch.onnx
+# This import monkey-patches graph manipulation methods on Graph, used for the
+# ONNX symbolics
+import torch.onnx.utils
+
+from torch.onnx.symbolic_helper import parse_args, _unimplemented, _black_list_in_opset
+import torch.onnx.symbolic_opset9
+
+
+# EDITING THIS FILE? READ THIS FIRST!
+# see Note [Edit Symbolic Files] in symbolic_helper.py
+
+# This file exports ONNX ops for opset 10
+# Opset 10 is supported by ONNX release 1.5.0
+# release on 04/24/19
+
+
+# Blacklist operators for this opset version.
+# These operators have been updated in ONNX but not re-implemented here.
+# It is very important to blacklist these operators to avoid exporting
+# models with mixed versions of operators.
+# TODO : add support for the blacklisted operators in black_listed_operators
+black_listed_operators = ["flip",
+                          "slice",
+                          "upsample_nearest2d", "upsample_bilinear2d",
+                          "avg_pool1d", "avg_pool2d", "avg_pool3d",
+                          "dropout", "feature_dropout", "alpha_dropout", "feature_alpha_dropout",
+                          "dropout_", "feature_dropout_", "alpha_dropout_", "feature_alpha_dropout_"]
+
+for black_listed_op in black_listed_operators:
+    vars()[black_listed_op] = _black_list_in_opset(black_listed_op)
+
+
+# Add new operator here
+@parse_args('v', 'i', 'i', 'i', 'i')
+def topk(g, self, k, dim, largest, sorted, out=None):
+    if out is not None:
+        _unimplemented("TopK", "Out parameter is not supported for topk")
+    if not largest:
+        _unimplemented("TopK", "Ascending TopK is not supported")
+    k = g.op("Constant", value_t=torch.tensor(k, dtype=torch.int64))
+    from torch.onnx.symbolic_opset9 import unsqueeze
+    k = unsqueeze(g, k, 0)
+    return g.op("TopK", self, k, axis_i=dim, outputs=2)
+
+
+def _max_pool(name, tuple_fn, ndims, return_indices):
+    @parse_args('v', 'is', 'is', 'is', 'is', 'i')
+    def symbolic_fn(g, input, kernel_size, stride, padding, dilation, ceil_mode):
+        if not stride:
```

Comment:
I moved the code for maxpool with dilation (and added ceil_mode).
But when I test with onnxruntime, I can see that there is a shape mismatch when dilation != {1} (with the old code as well).

(for example, I get a mismatch with Maxpool1d(kernel_size=3, stride=2, padding=0, dilatoin=2), and input=torch.randn(1,1,4)).


I will investigate the error and submit another PR with the fix if needed.